slack_bolt/adapter/wsgi/utils.py

```diff
@@ -0,0 +1 @@
+ENCODING = "utf-8"  # should always be utf-8
```

Comment:
This is fine for Slack apps but perhaps the comment should be clearer to clarify we understand this is not true for WSGI in general:
```suggestion
ENCODING = "utf-8"  # The content encoding for Slack requests/responses is always utf-8
```

Reply:
Do you think it's good to rename this file to "intenrals.py" for consistency with other source files in this project?

Reply:
Renamed the file to `internals.py` and updated the comment 💯